watchtower/wtwire/summary.go

```diff
@@ -0,0 +1,35 @@
+package wtwire
+
+import "fmt"
+
+// MessageSummary creates a human-readable description of a given Message. If
+// the type is unknown, an empty string is returned.
+func MessageSummary(msg Message) string {
```

Comment:
Can alternatively use the `MsgType()` method here. Although that would mean you still need a type assertion somewhere below.

Reply:
Using `MsgType()` would also work with non-pointer messages, while this won't?

Reply:
Why? Non-pointers can also implement interfaces.

Reply:
Indeed they can

> Using MsgType() would also work with non-pointer messages, while this won't?

The current version will work with non-pointer messages as well

Reply:
Realized that the non-pointer type actually doesn't implement the `Message` interface, so this is good :)